CHANGELOG.md

```diff
@@ -6,15 +6,50 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+
+## [4.0.0]
+
 ### Added
 
+#### Recipes
+ - bambi 0.14.0
```

Comment:
I think baton 2.1.0 also came in with this change

```suggestion
 - bambi 0.14.0
 - baton 2.1.0
```

Reply:
No, baton 2.1.0 is over a year old and was in the previous release.

Reply:
It is missing baton 3.0.0, though.

Reply:
Which I've added.